packages/app/src/runner/events/telemetry.ts

```diff
@@ -0,0 +1,59 @@
+import { telemetry } from '@packages/telemetry/src/browser'
+
+export const addTelemetryListeners = (getCypress) => {
+  const Cypress = getCypress()
+
+  Cypress.on('test:before:run', (attributes, test) => {
+    // we emit the 'test:before:run' events within various driver tests
```

Comment:
We don't need to gather telemetry on the odd test event that is firing off the weird events. Theres also no guarantee that the test also would close the span.

Reply:
I would still prefer to fix the invalid tests. If someone updates those tests in the future to pass a `test` they won't know they also need to close the span. We should just correct them now. If we don't want the telemetry for that type of scenario, we could add a way to temporarily disable telemetry instead of adding code in here that is not technically needed.

Reply:
But thats the thing, the test itself isn't invalid, it's testing exactly what it cares about. I don't think we should expect the test to accurately replicate 100% of fired events. I'd rather handle the rare odd case here.

Now i'm trying to remember which tests causes this issue....

Reply:
The test is invalid in that is doesn't conform to the defined [API](https://github.com/cypress-io/cypress/blob/89c23e08de76ff37093ac6c6958a8ffdaecc4fc0/cli/types/cypress.d.ts#L6240).

Reply:
We landed on doing a try catch here and leaving the weird tests.